spec.html

```diff
@@ -141,7 +141,7 @@ <h1>
 			1. Return _newError_.
 		</emu-alg>
 		<emu-note>
-			For implementations that provide implementation-specific stack for introspection APIs, the TypeError object can be augmented without releaving any information about the stack from within the ShadowRealm, and without providing direct access to any object reference from withtin the ShadowRealm.
+			For implementations that provide implementation-specific stack for introspection APIs, the TypeError object can be augmented without revealing any information about the stack from within the ShadowRealm, and without providing direct access to any object reference from withtin the ShadowRealm.
```

Comment:
😌

Reply:
duplicate of https://github.com/tc39/proposal-shadowrealm/pull/372#discussion_r966200757 but yes, not a "relief".